### PR TITLE
random file just as an example

### PR DESCRIPTION
**Pull request by davydw**
Hey @yusuke0127 

I have created an amazing file. I would love it to be to merged on the master branch

Could you have a look at my code ?

### CODE REVIEW

File: randomfile.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("ALLALALLALALA")
```
Comment (yusuke0127) [inline randomfile.py:1]: Fix this please!

Comment (davydw) [reply randomfile.py]: look at my new file, i think you should be satisfied with it !!!

**Review by yusuke0127 — APPROVED** [review]
LGTM!

**Comment by davydw** [comment]
# ERROR 1

This is what i did for problem !


# ERROR 2

ad;fjdlfj;dlfj;dljf;ldjf;lkdjfd

```python
def my function:
return whatever
```

```SQL
SELECT *
FROM TABLE
````

**Pull request merged**